src/about.nim

```diff
@@ -13,27 +13,34 @@ Options:
   --history     Search for matches in history
 """
 
-import docopt, sets, os, osproc, strutils
+import docopt, sets, os, osproc, strutils, sequtils
 import aboutpkg/fileutils
 
 let
-  args = docopt(doc, version = "about 0.1.0")
+  args = docopt(doc, version = "about 0.1.1")
   pattern = $args["<pattern>"]
   history = args["--history"]
 
 proc isExecutable(path: string): bool =
   let permissions = getFilePermissions(path)
   fpUserExec in permissions
 
+proc hasValidExtension(path: string): bool =
+  result = true
```

Comment:
default to true to preserve the original behavior for systems without the `PATHEXT` because there isn't a more rigorous test to see if a regular file with execute permissions is or isn't a program.